guides/app-guides/univ3-lp.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Details on Angle UniV3 incentive mechanism for LPs
+---
+
+# UniV3 LP incentives in Angle
+
+Angle implemented a specific mechanism to reward UniV3 liquidity providers according to their added value to the protocol. Basically, it allows to reward LPs more granularily according to the efficiency of the liquidity they provide. In turn, they have the opportunity to provide liquidity in more complex ways that benefit from all Uniswap V3 possibilities.
+
+You can find more details on the mechanism in [the original announcement](https://blog.angle.money/a-new-incentivization-mechanism-for-uniswap-v3-liquidity-8ce32fa611b1). Here, we will detail how the incentives are split and how you can provide liquidity on pools to earn ANGLE rewards.
```

Comment:
Let's put this in a hint

Reply:
And we need more context: In this guide, we detail how you can provide liquidity on the Uniswap V3 agEUR-ETH and agEUR-USDC pools on mainnet, and on the Uniswap V3 agEUR-USDC pool on Polygon, and earn rewards for that